lib/widgets/content.dart

```diff
@@ -38,91 +38,91 @@ import 'text.dart';
 class ContentTheme extends ThemeExtension<ContentTheme> {
```

Comment:
nit: perhaps it's intentional, but don't we usually use `content:` for the commit summary of changes like this?